feat: add create profile to onboarding

J-Son89 opened the pull request:
fixes: https://github.com/status-im/status-mobile/issues/15022

Allows user to set their custom color, user name and their profile picture.

For now it is using the old profile picture code but this will be updated in future work to a new design.

Additionally on ios there is a flickering issue which happens when the image is set:
https://github.com/ivpusic/react-native-image-crop-picker/issues/1658

To fix this we need to upgrade the underlying dependency and will get done outside of this pr.

**TO TEST**
Go to the new Generate Keys flow.
You should be then able to set your profile picture, user name and custom color and continue navigation until you generate keys and then land on the home page of the app.

Should be able to create a profile with or without a profile picture

J-Son89 committed:
https://github.com/status-im/status-go/compare/9c1c01c6...48eb7052

feat: add create profile to onboarding

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 47012a65 | [#1](https://ci.status.im/job/status-mobile/job/prs/job/tests/job/PR-15401/1/) | 2023-03-17 12:50:11 | ~5 min | `tests` | [:page_facing_up:`log`](https://status-im-mobile-prs.ams3.cdn.digitaloceanspaces.com/StatusIm-Mobile-230317-124508-47012a-pr15401-tests.log) |

du64 commented:
This top bar should match the translucent color below it.
referenced: #15362

![Screenshot_2023-03-17-08-31-14-92_65ad86f86fd2321a85272a99323da10f](https://user-images.githubusercontent.com/65834069/225918994-7d4f7306-4849-4715-9cee-2ca71811ad41.jpg)

cammellos commented on src/quo2/components/inputs/title_input/view.cljs:
```diff
@@ -16,7 +16,9 @@
-           default-value]
+           default-value
+           override-theme
+          ]
```
lint

J-Son89 replied:
done

status-im-auto commented:
## 94% of end-end tests have passed
```
Total executed tests: 33
Failed tests: 2
Passed tests: 31
```
```
IDs of failed tests: 702775,702838 
```
<h3>Failed tests (2)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=develop&PR_ID=15401&TR_CASE_IDS=702775,702838">Rerun failed tests</a></li><br/><h4>Class TestDeeplinkOneDeviceNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120712">test_deep_link_open_user_profile</a>, id: 702775 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: SignInButton </div><div>## Signed in successfully!</div></blockquote></p>

```
medium/test_deeplink_chat_share_profile.py:394: in test_deep_link_open_user_profile
    chat.wait_for_element_starts_with_text(ens_user['username'])
../views/base_view.py:465: in wait_for_element_starts_with_text
    return element.wait_for_element(wait_time)
../views/base_element.py:117: in wait_for_element
    raise TimeoutException(
 Device `1`: `Button` by` xpath`: `//*[starts-with(@text,'Legal Vibrant Indianabat')]` is not found on the screen after wait_for_element 
```

<b>[[Profile is often not opened in e2e builds for some reason. Needs to be investigated.]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b4fb4fc078f64e268a221e2e83e0c8d9?auth=1e03f728fbbb26060cc6e463ae30c894#99">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120721">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><p><blockquote><div>Device 2: Verifying that 'hello' is under today </div><div>Device 2: Looking for a message by text: hello </div></blockquote></p>

```
critical/test_public_chat_browsing.py:411: in test_community_message_send_check_timestamps_sender_username
    channel.verify_message_is_under_today_text(message, self.errors)
../views/chat_view.py:912: in verify_message_is_under_today_text
    message_element.wait_for_visibility_of_element()
../views/base_element.py:135: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: ChatElementByText by xpath:`//*[starts-with(@text,'hello')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[blocked by 14797]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d9dbd7c038745cb89ca875fe2093167?auth=c58dedf6f3330aae736a966bbf263da9#158">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0c3e7794ee04b3c8da0786a33ac7aa1?auth=243c20b899feda9065421fd8e6d5c54e#94">Steps, video, logs</a></li><li><a href="https://media.giphy.com/media/9M5jK4GXmD5o1irGrF/giphy.gif">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (31)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120708">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6d61d3f60ad4d27aa3e7f7ccc013898?auth=7bb762fe9e7af0b6501aae14799351a6#417">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56897f1b35794d1c947a3f049c7dcfb5?auth=bd5f6a041075942518f109233fd35a49#243">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120731">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6d61d3f60ad4d27aa3e7f7ccc013898?auth=7bb762fe9e7af0b6501aae14799351a6#400">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56897f1b35794d1c947a3f049c7dcfb5?auth=bd5f6a041075942518f109233fd35a49#201">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120710">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6d61d3f60ad4d27aa3e7f7ccc013898?auth=7bb762fe9e7af0b6501aae14799351a6#298">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56897f1b35794d1c947a3f049c7dcfb5?auth=bd5f6a041075942518f109233fd35a49#139">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120705">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6d61d3f60ad4d27aa3e7f7ccc013898?auth=7bb762fe9e7af0b6501aae14799351a6#85">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56897f1b35794d1c947a3f049c7dcfb5?auth=bd5f6a041075942518f109233fd35a49#71">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120714">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6d61d3f60ad4d27aa3e7f7ccc013898?auth=7bb762fe9e7af0b6501aae14799351a6#109">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56897f1b35794d1c947a3f049c7dcfb5?auth=bd5f6a041075942518f109233fd35a49#86">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120715">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6d61d3f60ad4d27aa3e7f7ccc013898?auth=7bb762fe9e7af0b6501aae14799351a6#472">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56897f1b35794d1c947a3f049c7dcfb5?auth=bd5f6a041075942518f109233fd35a49#316">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120706">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6d61d3f60ad4d27aa3e7f7ccc013898?auth=7bb762fe9e7af0b6501aae14799351a6#153">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56897f1b35794d1c947a3f049c7dcfb5?auth=bd5f6a041075942518f109233fd35a49#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120720">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6d61d3f60ad4d27aa3e7f7ccc013898?auth=7bb762fe9e7af0b6501aae14799351a6#357">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56897f1b35794d1c947a3f049c7dcfb5?auth=bd5f6a041075942518f109233fd35a49#175">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120716">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6d61d3f60ad4d27aa3e7f7ccc013898?auth=7bb762fe9e7af0b6501aae14799351a6#504">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/56897f1b35794d1c947a3f049c7dcfb5?auth=bd5f6a041075942518f109233fd35a49#364">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeeplinkOneDeviceNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120713">test_public_chat_open_using_deep_link</a>, id: 702776 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b4fb4fc078f64e268a221e2e83e0c8d9?auth=1e03f728fbbb26060cc6e463ae30c894#81">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120711">test_deep_link_with_invalid_user_public_key_own_profile_key</a>, id: 702774 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b4fb4fc078f64e268a221e2e83e0c8d9?auth=1e03f728fbbb26060cc6e463ae30c894#44">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120728">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ece3e619baac40cc81479353d3eb6883?auth=b1df5a9153321ab573f6fc915e706e20#60">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120709">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ece3e619baac40cc81479353d3eb6883?auth=b1df5a9153321ab573f6fc915e706e20#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120707">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6450c3d0e68349f89ee0e8b408e8f856?auth=b70ac1c3f5ba7a40f8705cec3e59fa2b#178">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9650af80c1744d35a1f7650e97563375?auth=d0e3649bd30431607c24adc5a50d39d3#110">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7535c2aadd904e63a967512809b5c5fd?auth=90ef3ad3a887d71e8073b9ab81eaadaa#89">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120718">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6450c3d0e68349f89ee0e8b408e8f856?auth=b70ac1c3f5ba7a40f8705cec3e59fa2b#119">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9650af80c1744d35a1f7650e97563375?auth=d0e3649bd30431607c24adc5a50d39d3#65">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7535c2aadd904e63a967512809b5c5fd?auth=90ef3ad3a887d71e8073b9ab81eaadaa#65">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120719">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6450c3d0e68349f89ee0e8b408e8f856?auth=b70ac1c3f5ba7a40f8705cec3e59fa2b#133">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/9650af80c1744d35a1f7650e97563375?auth=d0e3649bd30431607c24adc5a50d39d3#90">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7535c2aadd904e63a967512809b5c5fd?auth=90ef3ad3a887d71e8073b9ab81eaadaa#70">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120735">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a785d9fe8d5f479884ce128f3a71eaf6?auth=dacdbbaea2b090a9438afc22dffd74bf#169">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32031e7a36124d978fb7503545ed5234?auth=efba04182700b5cda3ea71a55495e568#93">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120737">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a785d9fe8d5f479884ce128f3a71eaf6?auth=dacdbbaea2b090a9438afc22dffd74bf#309">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32031e7a36124d978fb7503545ed5234?auth=efba04182700b5cda3ea71a55495e568#129">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120734">test_navigation_jump_to</a>, id: 702936 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a785d9fe8d5f479884ce128f3a71eaf6?auth=dacdbbaea2b090a9438afc22dffd74bf#148">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32031e7a36124d978fb7503545ed5234?auth=efba04182700b5cda3ea71a55495e568#90">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120736">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a785d9fe8d5f479884ce128f3a71eaf6?auth=dacdbbaea2b090a9438afc22dffd74bf#275">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/32031e7a36124d978fb7503545ed5234?auth=efba04182700b5cda3ea71a55495e568#109">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120723">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d9dbd7c038745cb89ca875fe2093167?auth=c58dedf6f3330aae736a966bbf263da9#213">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0c3e7794ee04b3c8da0786a33ac7aa1?auth=243c20b899feda9065421fd8e6d5c54e#133">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120726">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d9dbd7c038745cb89ca875fe2093167?auth=c58dedf6f3330aae736a966bbf263da9#239">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0c3e7794ee04b3c8da0786a33ac7aa1?auth=243c20b899feda9065421fd8e6d5c54e#148">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120717">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d9dbd7c038745cb89ca875fe2093167?auth=c58dedf6f3330aae736a966bbf263da9#398">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0c3e7794ee04b3c8da0786a33ac7aa1?auth=243c20b899feda9065421fd8e6d5c54e#243">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120733">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d9dbd7c038745cb89ca875fe2093167?auth=c58dedf6f3330aae736a966bbf263da9#294">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0c3e7794ee04b3c8da0786a33ac7aa1?auth=243c20b899feda9065421fd8e6d5c54e#213">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120727">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d9dbd7c038745cb89ca875fe2093167?auth=c58dedf6f3330aae736a966bbf263da9#415">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0c3e7794ee04b3c8da0786a33ac7aa1?auth=243c20b899feda9065421fd8e6d5c54e#266">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120724">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d9dbd7c038745cb89ca875fe2093167?auth=c58dedf6f3330aae736a966bbf263da9#271">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0c3e7794ee04b3c8da0786a33ac7aa1?auth=243c20b899feda9065421fd8e6d5c54e#205">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120722">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d9dbd7c038745cb89ca875fe2093167?auth=c58dedf6f3330aae736a966bbf263da9#193">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0c3e7794ee04b3c8da0786a33ac7aa1?auth=243c20b899feda9065421fd8e6d5c54e#104">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120725">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1d9dbd7c038745cb89ca875fe2093167?auth=c58dedf6f3330aae736a966bbf263da9#171">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d0c3e7794ee04b3c8da0786a33ac7aa1?auth=243c20b899feda9065421fd8e6d5c54e#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120732">test_activity_center_contact_request_cancel_outgoing_no_pn</a>, id: 702871 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c95f8dc506b04681b14ceb9a394d0f63?auth=94f3daa33e5ff21a35deebf8b7d8f7fc#54">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4c50bb16ade94e54878ace042d416af4?auth=75506410d368f00b46ebc9f9fb684790#43">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120729">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c95f8dc506b04681b14ceb9a394d0f63?auth=94f3daa33e5ff21a35deebf8b7d8f7fc#68">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4c50bb16ade94e54878ace042d416af4?auth=75506410d368f00b46ebc9f9fb684790#83">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3120730">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c95f8dc506b04681b14ceb9a394d0f63?auth=94f3daa33e5ff21a35deebf8b7d8f7fc#96">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4c50bb16ade94e54878ace042d416af4?auth=75506410d368f00b46ebc9f9fb684790#110">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

churik commented:
### ISSUE 1: profile image is not set in the profile if it was set during "Generate keys" flow

**Steps:**
1) "Generate new key"
2) Set profile image, name, color
3) Finish flow and check the user profile

**Expected result:**
can see user picture

**Actual result:**
![IMAGE 2023-03-21 13:34:18](https://user-images.githubusercontent.com/4557972/226607275-5a8db281-8b16-4508-9fbe-f524060b26fd.jpg)

**OS:** IOS, Android


### ISSUE 2: White top bar on IOS
The issue is not reproducible on nightly 21/03/23
<img width="230" alt="Monosnap 2023-03-21 13-35-18" src="https://user-images.githubusercontent.com/4557972/226607580-eda65152-48fa-40ee-88e0-a987ec665b6d.png">

**OS:** IOS 16 (Iphone mini)
 
 
 ### Question 3:  https://github.com/status-im/status-mobile/issues/15362 will be fixed apart, right?

Parveshdhull commented on src/quo2/foundations/colors.cljs:
```diff
@@ -236,9 +236,12 @@
+                                     (if (theme/dark?) :yang :yin)
+                                     (keyword color)) suffix])]
         (if opacity (alpha base-color (/ opacity 100)) base-color))))))
-
```
Please add line

Parveshdhull commented on src/status_im/multiaccounts/login/core.cljs:
```diff
@@ -479,8 +479,14 @@
-  (if (get db :tos/accepted?)
+  (cond
+    (get db :onboarding-2/new-account?)
+    (re-frame/dispatch [:navigate-to :enable-notifications])
```
@smohamedjavid this pr addresses this flow, so can also be used for sign in too.

Parveshdhull commented on src/status_im2/contexts/onboarding/create_password/view.cljs:
```diff
@@ -32,7 +33,9 @@
-     {:on-press #(rf/dispatch [:navigate-to :enable-biometrics])
+     {:on-press #(rf/dispatch
+                  [:onboarding-2/password-set
+                   (security/mask-data "test123")])
```
leftover?

J-Son89 replied:
no this is just placeholder until @ulisesmac pr for password page is merged

Parveshdhull commented on src/status_im2/contexts/onboarding/create_profile/view.cljs:
```diff
@@ -1,12 +1,18 @@
 
+(def default-color :blue)
+
+(def max-length 24)
```
Should we move them to constants file?

Parveshdhull commented on src/status_im2/contexts/onboarding/create_profile/view.cljs:
```diff
@@ -1,12 +1,18 @@
+            [status-im2.contexts.onboarding.select-photo.method-menu.view :as method-menu]
             [utils.re-frame :as rf]))
 
+(def default-color :blue)
```
At most places we have :default color as :primary, but it looks like figma colors file don't even have primary color anymore 
We probably need to change this too on all the places. (probably in separate PR)

J-Son89 replied:
probably this should just be in constants or something like that.

Parveshdhull commented on src/status_im2/contexts/onboarding/generating_keys/view.cljs:
```diff
@@ -0,0 +1,29 @@
+    [quo/text
+     {:size   :heading-1
+      :weight :semi-bold
+      :style  {:color colors/white}} "Generating keys..."]]])
```
move to translations file

Parveshdhull reviewed (approved):
some minor suggestions, apart from them lgtm

briansztamfater commented on src/status_im2/subs/root.cljs:
```diff
@@ -268,6 +268,10 @@
 
+; onboarding
+
+(reg-root-key-sub :onboarding-2/profile :onboarding-2/profile)
```
Could we use `onboarding/profile` instead of `onboarding-2`? I feel that we will need / want to change `onboarding-2/...` in the future and maybe we can do it now

J-Son89 replied:
this is intentional to have a clear separation of the old onboarding and new onboarding subs. The idea being that when we move away from the old onboarding that we can completely nuke all `onboarding/` subs etc and rename all new `onboarding-2` subs to `onboarding`. 
Sure we could do it now by changing the old to something like `onboarding-old/` instead but this would be a lot of extra changes, noise and work and potentially brings in bugs from missing some parts.
@cammellos and I agreed upon this. 
Can be changed of course but imo it's the better option as is 👍

briansztamfater replied:
Understood. I did a quick and vague search I haven't seen other `onboarding/` subs or db keys so that's why I don't see necessary to use the `-2` suffix, but it is okay for me anyway

J-Son89 commented:
> ### ISSUE 1: profile image is not set in the profile if it was set during "Generate keys" flow
> **Steps:**
> 
> 1. "Generate new key"
> 2. Set profile image, name, color
> 3. Finish flow and check the user profile
> 
> **Expected result:** can see user picture
> 
> **Actual result:** ![IMAGE 2023-03-21 13:34:18](https://user-images.githubusercontent.com/4557972/226607275-5a8db281-8b16-4508-9fbe-f524060b26fd.jpg)
> 
> **OS:** IOS, Android
> 
> ### ISSUE 2: White top bar on IOS
> The issue is not reproducible on nightly 21/03/23 <img alt="Monosnap 2023-03-21 13-35-18" width="230" src="https://user-images.githubusercontent.com/4557972/226607580-eda65152-48fa-40ee-88e0-a987ec665b6d.png">
> 
> **OS:** IOS 16 (Iphone mini)
> 
> ### Question 3: #15362 will be fixed apart, right?

@churik I have fixed Issue 1 on ios and android 👍 Issue 2 will be handled separate to this pr as it's not in scope and Question 3 will also be handled separately. 👍

churik commented:
### ISSUE 1: cannot read property 'toUpperCase' in case of starting if the name is starting from `   D` (any uppercase with space at the beginning)

**Steps:**
1) "Generate new key"
2) on the name filed type space and uppercase, i.e. `    F`
**Expected result:**
can proceed
**Actual result:**


https://user-images.githubusercontent.com/4557972/226680347-79762fd0-1f11-4615-9801-df77482eaed5.mp4


![IMAGE 2023-03-21 17:41:04](https://user-images.githubusercontent.com/4557972/226680256-f0a0846a-dd55-44ae-88e0-d59776ecc3eb.jpg)


**OS:** IOS, Android

J-Son89 commented:
> ### ISSUE 4: cannot read property 'toUpperCase' in case of starting if the name is starting from any uppercase with space at the beginning
> **Steps:**
> 
> 1. "Generate new key"
> 2. on the name filed type space and uppercase, i.e. `    F`
>    **Expected result:**
>    can proceed
>    **Actual result:**
> 
> ![IMAGE 2023-03-21 17:41:04](https://user-images.githubusercontent.com/4557972/226680256-f0a0846a-dd55-44ae-88e0-d59776ecc3eb.jpg)
> 
> **OS:** IOS, Android

@churik, I fixed this whitespace issue and added some basic validation to the profile name input as per the designs too 👍

churik commented:
@J-Son89 

### ISSUE 5 Font is too white and invisible in black/white color


**OS:** IOS, Android
![IMAGE 2023-03-22 12:43:00](https://user-images.githubusercontent.com/4557972/226893898-c3c44106-d358-4bf5-ae2c-b3e968123369.jpg)

![IMAGE 2023-03-22 12:42:56](https://user-images.githubusercontent.com/4557972/226893885-4d51abfe-3e00-40a7-93c8-2a50da18c1cf.jpg)

### ISSUE 6 Difference between design and actual implementation 
<img width="432" alt="photo_2023-03-22 12 48 42 jpeg 2023-03-22 12-49-18" src="https://user-images.githubusercontent.com/4557972/226896606-fc380b99-470e-4bf8-91da-6aab32479f2b.png">

- 6.1 The placeholder font color is not aligned
- 6.2 The cursor color is not aligned
- 6.3 No opacity either on IOS or on Android
- 6.4 'Continue' button is overlapped by the keyboard and the view is not scrollable 

### ISSUE 7 Button continue is still available when validation is not passed
Therefore, can create multi-account with invalid names.

**Steps:**
- set name with emoji 
- finish the flow

Besides, there is no specification of what is considered 'generic' names. Per design  from https://www.figma.com/file/o4qG1bnFyuyFOvHQVGgeFY/Onboarding-for-Mobile?node-id=2241%3A250700&t=fD9St3RpvL2wULMb-0 it is only `Ethereum`, but I guess it is not enough. But it can be addressed separately.

Let me know if smth from mentioned is already discussed and if I need to ping designers about acceptable differences for now.

Thank you!

J-Son89 commented:
@churik, is it possible I follow up addressing those issues? Getting this pr in will unblock some of the developers on the other pages.

To answer each item:
ISSUE 5 Font is too white and invisible in black/white color
- Design team have not given us a clear explanation of how this is to be handled for black and white yet so we are unable to build it correctly. IMO it is probably better that we remove this option, will this discuss with design team but for now it will not be working correctly as long as it is selectable.

ISSUE 6 & 7 should be addressed immediately but I can do it in a follow up issue/pr.

Validation is only general at this point as we do not have an extensive list of common names etc that we need to remove. 

cc: @cammellos

churik commented:
I'll open follow-up, no worries! thank you for your work, ready to be merged
